prow/cmd/peribolos/OWNERS

```diff
@@ -2,7 +2,9 @@
 
 reviewers:
 - cjwagner
+- sig-contributor-experience-technical-leads
```

Comment:
We should really be adding people that have worked on this, which happens to be the current contribex leads.

but granting people sudden broad /approve opaquely by way of updating an alias like this is an anti-pattern I will continue to discourage, I don't think we should have ever added the feature